docs/pipelines/tasks/deploy/azure-resource-group-deployment.md

```diff
@@ -53,6 +53,39 @@ Use this task to deploy, start, stop, and delete Azure Resource Groups.
 |`deploymentOutputs`<br/>Deployment outputs|(Optional) Provide a name for the variable for the output variable which will contain the outputs section of the current deployment object in string format. You can use the **ConvertFrom-Json** PowerShell cmdlet to parse the JSON object and access the individual output values.|
 |`addSpnToEnvironment`<br/>Access service principal details in override parameters| Adds service principal ID and key of the Azure endpoint you chose to the script's execution environment. You can use these variables: **$servicePrincipalId** and **$servicePrincipalKey** in your override parameters like **-key $servicePrincipalKey**|
 
+## FAQs
+
+## Error: Internal Server Error
+
+These issues are mostly transient in nature. There are multiple reasons why it could be happening:
+1. One of the Azure service you're trying to deploy is down in that region
+2. Azure Pipelines service itself is going through maintenance. Keep an eye out on https://status.dev.azure.com/ for downtimes.
+
+However, we've seen some instances where this is due to an error in the ARM template, such as:
+* Azure service you're trying to deploy doesn't support the region you've chosen for the resource.
+
+## Error: Timeout
+
+Timeout issues could be coming from two places:
+1. Azure Pipelines Agent
+2. Portal Deployment
+
+You can identify if the timeout is from portal, by checking for the portal deployment link that'll be in the console. If there's no link, this is likely due to Azure Pipelines agent. If there's a link, follow the link to see if there's a timeout that has happened in the portal deployment.
```

Comment:
I am not sure whether users will understand `console` here. Better say `task log`